numberformat/proposed.emu

```diff
@@ -111,13 +111,17 @@
             1. Let _mxfdDefault_ be 0.
           1. Else,
             1. Let _mxfdDefault_ be 3.
+        1. Let _roundingIncrement_ be ? GetNumberOption(_options_, *"roundingIncrement"*, 1, 5000, 1).
+        1. If _roundingIncrement_ is in « 1, 2, 5, 10, 20, 25, 50, 100, 200, 250, 500, 1000, 2000, 2500, 5000 », then
```

Comment:
Issue: The default value of `_roundingIncrement_` is 1. We only want to override `_mxfdDefault_` only if there was an explicit setting.